### PR TITLE
Update to GraalVM 20.1.0

### PR DESCRIPTION
**Pull request by wirthi**
Updating the values from GraalVM 20.0.0 to 20.1.0.

### CODE REVIEW

File: data-esnext.js [hunk 1]
```diff
@@ -1585,7 +1585,7 @@ exports.tests = [
     safari13: true,
     safaritp: true,
     graalvm19: false,
-    graalvm20: {val: 'flagged', note_id: "graalvm-es2020", note_html: "The feature have to be enabled via <code>--js.ecmascript-version=2020</code> flag"},
```
Comment (ljharb) [inline data-esnext.js:1588]: it seems better to me to keep this as-is, and add a new value for graalvm 20.1.

**Review by chicoxyzzy — COMMENTED** [review]
I think that it would be better to keep graalvm20 and add graalvm20_1

**Comment by wirthi** [comment]
Why would you keep the old version, what's the advantage? From our point of view, 20.0 is superseded, no-one should be using it any longer. At the end of the year, we would have 19.3.x, 20.0, 20.1, 20.2, and 20.3 - and add four more next year. When do we delete them again?

IMHO, there should be our 19.3 LTS version, and there should be the most recent version of 20 (20.1 currently).

**Comment by ljharb** [comment]
@wirthi people use old versions of everything, what they "should" be using isn't the point. You could certainly mark 20.0 as "very obsolete", but that doesn't mean the test data for it should be erased.

In other words, I don't think test results for any release should be deleted *ever* - 100 years from now, I should be able to determine exactly how much v20.0 complied with, or violated, the spec.

**Comment by wirthi** [comment]
Ok, I am not convinced this is necessary/reasonable, but if that's how you do things here, I'll set it up like that.

Is it possible to NOT duplicate all the values? I see there is an `equals` entry in `environments.json` - can I use that for this case, i.e. have the data in `graalvm20` and only specify exceptions for `graalvm20_1`?

-- Christian

**Comment by ljharb** [comment]
@wirthi yep, exactly! you'd set up each new graal version as basically inheriting from the previous one, and then you'd only have to add the results that differ.

**Review by chicoxyzzy — APPROVED** [review]

**Comment by wirthi** [comment]
I have rebased to the current state of gh-pages. Ready to be merged from my side.

**Comment by chicoxyzzy** [comment]
LGTM, thanks!

**Pull request merged**